Assets/Scripts/Map/TileData.cs

```diff
@@ -24,11 +24,25 @@ public int R
 	}
 	private int r;
 
-	public TileData(Tile tile, Player player, int tileID, int q, int r)
+	public string TileName
+	{
+		get { return tileName; }
+	}
+	private string tileName;
+
+	public string Description
+	{
+		get { return desc; }
+	}
+	private string desc;
+
+	public TileData(Tile tile, Player player, int tileID, string name, string desc, int q, int r)
```

Comment:
Instead of passing the `name` and `description` separately, just pass the `TileLocalization` object and store it in this class instead. This way, the constructor isn't unnecessarily long. and we don't have to update this file if `TileLocalization.cs` changes.